internal/guest/runtime/hcsv2/uvm.go

```diff
@@ -467,6 +467,24 @@ func (h *Host) SignalContainerProcess(ctx context.Context, containerID string, p
 		return err
 	}
 
+	signalingInitProcess := (processID == c.initProcess.pid)
```

Comment:
we discussed this offline and will revisit this use case later. With current implementation, old behavior shouldn't change.